backend/package.json

```diff
@@ -14,7 +14,6 @@
   "dependencies": {
     "cors": "^2.8.5",
     "@sendgrid/mail": "^7.1.0",
-    "cors": "^2.8.5",
```

Comment:
Is dit de bedoeling??

Reply:
Jup, hiervoor stond hij er dubbel in. Check 2 regels erboven